src/gsi/setupbend.f90

```diff
@@ -579,290 +580,295 @@ subroutine setupbend(obsLL,odiagLL, &
 
      if(ratio_errors(i) > tiny_r_kind)  then ! obs inside model grid
 
-        if (alt <= six) then
-           if (top_layer_SR >= 1) then ! SR exists for at least one layer. Check if obs is inside 
-              if (tpdpres(i)==ref_rad(top_layer_SR+1)) then !obs inside model SR layer
-                 qcfail(i)=.true.
-              elseif (tpdpres(i) < ref_rad(top_layer_SR+1)) then !obs below model close-to-SR layer
-                 qcfail(i)=.true.
-              elseif (tpdpres(i) >= ref_rad(top_layer_SR+1) .and. tpdpres(i) <= ref_rad(top_layer_SR+2)) then !source too close
-                 qcfail(i)=.true.
-              else !above
-              endif
-           endif
-
-!          check for SR in obs, will be updated in genstats. 
-           if ( data(igps,i) >= 0.03 .and. qc_layer_SR) then
-              kprof = data(iprof,i)
-              toss_gps_sub(kprof) = max (toss_gps_sub(kprof),data(igps,i))
-           endif
-        endif
-
-!       get pressure (in mb), temperature and moisture at obs location
-        call tintrp31(ges_lnprsi(:,:,1:nsig,:),dpressure,dlat,dlon,hob,&
+       if (alt <= six) then
+          if (top_layer_SR >= 1) then ! SR exists for at least one layer. Check if obs is inside 
+             if (tpdpres(i)==ref_rad(top_layer_SR+1)) then !obs inside model SR layer
+                qcfail(i)=.true.
+             elseif (tpdpres(i) < ref_rad(top_layer_SR+1)) then !obs below model close-to-SR layer
+                qcfail(i)=.true.
+             elseif (tpdpres(i) >= ref_rad(top_layer_SR+1) .and. tpdpres(i) <= ref_rad(top_layer_SR+2)) then !source too close
+                    qcfail(i)=.true.
```

Comment:
Please remove the extra indentation on line 590.

Reply:
Indentation removed on line 590 at   d199ed5.    

Add print statement in the empty else branch of `src/gsi/setupbend.f90`.  Build `global_gsi.x` and run executable for four gfs cases:  2021053112 to 2021060106.   Each case generated printout from the empty else branch, on average 13621 entries per outer loop.   Removed the else branch from `src/gsi/setupbend.f90`.  Build `global_gsi.x` and reran the same four gfs cases.  The resulting analyses are identical.    This is not surprising.   The empty else branch currently does nothing.   Removing the empty else branch should not alter results.   

Two questions remain
1. Why is the else statement in the code?
2. What code, if any, should be in the empty else branch

Kristen asked Hui these questions.  Both Kristen and Hui agree the empty else branch may be removed.   

Remove empty else branch at d199ed5.